lib/flatpak-game.lua

```diff
@@ -351,7 +351,7 @@ function save_manifest(metadata, enable_network)
 	-- http://flatpak.org/developer.html#Anatomy_of_a_Flatpak_App
 	write_line(f, '[Application]')
 	write_line(f, 'name=' .. metadata.id)
-	write_line(f, 'runtime=org.freedesktop.Platform/' .. metadata.arch .. '/1.4')
+	write_line(f, 'runtime=org.freedesktop.Platform/' .. metadata.arch .. '/1.6')
```

Comment:
Why? I don't think this is a good idea, especially when the openssl soname was recently changed in it. The runtime was selected to be as compatible as possible with Ubuntu LTS, this change would break that.